packages/components/package.json

```diff
@@ -47,6 +47,7 @@
     "ember-a11y-refocus": "^4.1.3",
     "ember-cli-sass": "^11.0.1",
     "ember-composable-helpers": "^5.0.0",
+    "ember-concurrency": "^4.0.2",
```

Comment:
note: all products using `ember-concurrency` are on 4.x

Reply:
Ok thanks, should I change it to "^4.0.0"?

Reply:
`4.0.2` is fine, it's the latest. yarn will decide the exact version in the resolution process

Reply:
Updated. Does it look ok the way I formatted it?